nms/app/packages/magmalte/app/components/main/Index.js

```diff
@@ -83,21 +84,23 @@ export default function Index() {
   return (
     <NetworkContext.Provider value={{networkId, networkType}}>
       <LteContextProvider networkId={networkId} networkType={networkType}>
-        <div className={classes.root}>
-          <AppSideBar
-            mainItems={[<SectionLinks key={1} />, <VersionTooltip key={2} />]}
-            secondaryItems={[<NetworkSelector key={1} />]}
-            projects={getProjectLinks(tabs, user)}
-            showSettings={shouldShowSettings({
-              isSuperUser: user.isSuperUser,
-              ssoEnabled,
-            })}
-            user={user}
-          />
-          <AppContent>
-            <SectionRoutes />
-          </AppContent>
-        </div>
+        <FEGContextProvider networkId={networkId} networkType={networkType}>
```

Comment:
Semantically, it seems a little strange to me for the `FEGContextProvider` to be nested inside the `LteContextProvider`. I think it makes sense to provide a `NetworkContextProvider` that returns either one, though that might be too much extra code.

Reply:
I did that, but yes there is too much extra code. However, it is semantically correct now.

Reply:
Also removed federation category.